index.html

```diff
@@ -11,140 +11,142 @@
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item active">
+              <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="statistics.html">Statistics</a>
+            </li>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Get Involved
+              </a>
+              <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                <a class="dropdown-item" href="#">Invest</a>
+                <a class="dropdown-item" href="#">Donate Data</a>
+                <a class="dropdown-item" href="#">Career Opportunities</a>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+
+      
+      <div class="heading">
+        <h1 id="logo">Gradflow</h1>
+        <!-- <img id="heading-img" src="images/change-close-up-coins-210698.jpg"> -->
+        <h2 id="description">The app analyzing the true impact of higher education on lifetime earnings potential. <br>Know where you're headed.</h2>
       </div>
-    </nav>
 
-    <h2>An app analyzing the true value of higher education over the course of one's life.</h2>
-<!-- Carousel Starts Here -->
-    <div class="bd-example">
-      <div id="carouselExampleCaptions" class="carousel slide" data-ride="carousel">
-        <ol class="carousel-indicators">
-          <li data-target="#carouselExampleCaptions" data-slide-to="0" class="active"></li>
-          <li data-target="#carouselExampleCaptions" data-slide-to="1"></li>
-          <li data-target="#carouselExampleCaptions" data-slide-to="2"></li>
-        </ol>
-        <div class="carousel-inner">
-          <div class="carousel-item active">
-            <img src="images/academy-celebrate-celebration-267885.jpg" class="d-block w-100" alt="...">
-            <div class="carousel-caption d-none d-md-block">
-              <h5>From Graduate to Professional</h5>
-              <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
+      <!-- Carousel Starts Here -->
+      <div class="bd-example">
+        <div id="carouselExampleCaptions" class="carousel slide" data-ride="carousel">
+          <ol class="carousel-indicators">
+            <li data-target="#carouselExampleCaptions" data-slide-to="0" class="active"></li>
+            <li data-target="#carouselExampleCaptions" data-slide-to="1"></li>
+            <li data-target="#carouselExampleCaptions" data-slide-to="2"></li>
+          </ol>
+          <div class="carousel-inner">
+            <div class="carousel-item active">
+              <img src="images/academy-celebrate-celebration-267885.jpg" class="d-block w-100" alt="...">
```

Comment:
I recommend using the *alt* attribute for better user accessibility such as screen readers. Also, if image fails to load, the alt description will serve as backup text.